0007634 - contact annuaire ajouter un champ pour affichage des autres emails

CottonDamien opened the pull request:
- Création d’un nouvel onglet dans l’interface
- Affichage de l’adresse mail principale
- Ajout de l’affichage des autres adresses de messagerie associées

Rotomeca reviewed (changes requested):
Le nom du commit n'est pas bon, ce n'est pas "wsp" mais "contact" plutôt.
A faire : 
- undo last commit
- Changer le nom
- commit
- git push --force

Rotomeca commented:
![image](https://github.com/user-attachments/assets/f32de1e5-bf35-4011-8c73-d0e4ffa5bd30)
Problème d'interface

Commit:
feat(contact): Ajout d'un onglet pour afficher les adresses email principales et secondaires

- Création d’un nouvel onglet dans l’interface
- Affichage de l’adresse mail principale
- Ajout de l’affichage des autres adresses de messagerie associées
